pkg/client/clientcmd/loader.go

```diff
@@ -33,31 +33,35 @@ import (
 const (
 	RecommendedConfigPathFlag   = "kubeconfig"
 	RecommendedConfigPathEnvVar = "KUBECONFIG"
```

Comment:
> Recommended* consts for the file names in local and home dirs (.kubeconfig and /.kube/.kubeconfig)?

I'd prefer not to do that in this pull.  The location of the files isn't necessarily fixed.  https://github.com/GoogleCloudPlatform/kubernetes/issues/4615 proposes new locations (for good reasons) and https://github.com/GoogleCloudPlatform/kubernetes/issues/4428 proposes changes as well.  I don't think we've closed on a recommended location or even whether that location is a file or directory.